org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/DocumentLifeCycleHandlerTest.java

```diff
@@ -117,15 +117,28 @@ public void testUnimplementedMethods() throws Exception {
 		ICompilationUnit cu = pack1.createCompilationUnit("F.java", buf.toString(), false, null);
 		openDocument(cu, cu.getSource(), 1);
 
-		buf = new StringBuilder();
+		List<Command> commands = getCodeActions(cu);
+		assertTrue(commands.size() == 1);
+	}
+
+	@Test
+	public void testRemoveDeadCodeAfterIf() throws Exception {
```

Comment:
this test doesn't check for the range is multi-line, i.e what we're trying to fix in the 1st place